### PR TITLE
ignore package-lock & test node 12

### PR DESCRIPTION
**Pull request by gregory**
here we go @mcollina

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function traceCaller (pinoInstance) {
 
   function asJson (...args) {
     args[0] = args[0] || Object.create(null)
-    args[0].caller = Error().stack.split('\n')[STACKTRACE_OFFSET].substr(LINE_OFFSET)
+    args[0].caller = Error().stack.split('\n').filter(s => !s.includes('node_modules/pino'))[STACKTRACE_OFFSET].substr(LINE_OFFSET)
```
Comment (phra) [inline index.js]: this is exactly as #18

Comment (gregory) [reply index.js]: If #18 get merged before then i'll rebase, but this is a requirement for the tests to pass.

File: .travis.yml [hunk 2]
```diff
@@ -4,5 +4,6 @@ node_js:
 - '6'
 - '8'
 - '10'
+- '12'
```
Comment (phra) [inline .travis.yml]: this was superseded by #21

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 node_modules
+package-lock.json
```
Comment (phra) [inline .gitignore]: see https://stackoverflow.com/a/44210813/8006810

**Pull request closed without merging**